packages/backend/package.json

```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "build": "npm-run-all clean compile webpack-dev frontend:all",
     "bundle": "webpack --mode production",
-    "ci": "npm-run-all clean compile copy:test-resources coverage bundle frontend:all package",
+    "ci": "npm-run-all clean compile copy:test-resources coverage bundle frontend:copy package coverage:copy",
```

Comment:
you replaced `frontend:all` with  `frontend:copy` which doesn't build the frontend. Are we sure the frontend build always exist? Isn't better to keep the `frontend:all` to make sure we have the `dist` in hands before we copy it to the backend?